src/libraries/System.Private.CoreLib/src/System/Enum.cs

```diff
@@ -315,13 +315,43 @@ public static Type GetUnderlyingType(Type enumType)
             (TEnum[])GetValues(typeof(TEnum));
 #endif
 
-        [RequiresDynamicCode("It might not be possible to create an array of the enum type at runtime. Use the GetValues<TEnum> overload instead.")]
+        [RequiresDynamicCode("It might not be possible to create an array of the enum type at runtime. Use the GetValues<TEnum> overload or the GetValuesAsUnderlyingType method instead.")]
         public static Array GetValues(Type enumType)
         {
             ArgumentNullException.ThrowIfNull(enumType);
             return enumType.GetEnumValues();
         }
 
+        /// <summary>
+        /// Gets an array of values of the underlying type of the Enum.
```

Comment:
This should use the same wording style as the existing methods https://docs.microsoft.com/en-us/dotnet/api/system.enum.getvalues